.perlcriticrc

```diff
@@ -60,3 +60,9 @@ max_nests = 12
 max_elsif = 13
 [Variables::ProhibitUnusedVariables]
 # /end severity 3
+
+
+# severity 1
+[Documentation::PodSpelling]
+stop_words = JSON JS ecoscore api param json jsonp xml url TODO TAGID Nutri barcode Folksonomy mongoDB OpenFoodFacts eval CSV UTF FILEHANDLE STDOUT iso GeoIP geolocation ip eg po gettext Crowdin hashref https http submodules Config submodule lc lang barcodes packagings HARDCODED hardcoded hashrefs scrypt Scrypt md IPs csv ssconvert openfoodfacts couvercle métal FSC Française VPF viande de porc française Porc Français Origine français Pâtes aromatisées fraise canneberge l'acérola Ingrédients Pulpe poire sucre sirop blé céléri GDSN Equadis CodeOnline nodejs gtin csv CIN CIC NGO EREN PNNS et Santé matche undef NaN kcal AGS saturés barquette de CIQUAL AgriBalyse LCA EPI nutriscore msgctxt userid UUID appid uuid FZF RTFSG openfoodfacts subfield Nutriscore Santé publique kJ colza Valeur EXIF PNG incrontab gif jpeg jpf png heic Tesseract Robotoff jpg tesseract sprintf nutri Nutri www formatter ur Storable uncharacterized Agribalyse malus Canonicalizes canonicalized Programme Envol GSDN Carrefour multiline Multiline backticks scanbot taxonomize taxonomized gnumeric sto tagid canonicalize str Lowercased stopwords greated rescale overriden contenant maltodextrine vitamines stabilisant glycérol gélifiant pectine acidifiant acide citrique arôme naturel poudre d'acérola acérola Fabriqué dans un utilisant Allergènes flavouring flavourings dont Catalogue
```

Comment:
Maybe `stop_words_file` for such a long list of words?

Reply:
How's this?

Reply:
LGTM